content/master/concepts/patch-and-transform.md

```diff
@@ -0,0 +1,1692 @@
+  name: my-example-claim-978mh
+  resourceVersion: "40367"
+  uid: d64f3020-4f9d-4c52-bd14-f8491869d2c5
+spec:
+  claimRef:
+    apiVersion: example.org/v1alpha1
+    kind: ExampleClaim
+    name: my-example-claim
+    namespace: default
+  compositionRef:
+    name: example-composition
+  compositionRevisionRef:
+    name: example-composition-f4c519a
+  compositionUpdatePolicy: Automatic
+  desiredRegion: eu-north-1
+  field1: field1-text
+  field2: field2-text
+  resourceRefs:
+  - apiVersion: s3.aws.upbound.io/v1beta1
+    kind: Bucket
+    name: my-example-claim-978mh-r6z64
+  - apiVersion: s3.aws.upbound.io/v1beta1
+    kind: Bucket
+    name: my-example-claim-978mh-cnlhj
+  - apiVersion: s3.aws.upbound.io/v1beta1
+    kind: Bucket
+    name: my-example-claim-978mh-rv5nm
+status:
+  conditions:
+  - lastTransitionTime: "2023-07-14T17:36:30Z"
+    reason: ReconcileSuccess
+    status: "True"
+    type: Synced
+  - lastTransitionTime: "2023-07-14T17:36:55Z"
+    reason: Available
+    status: "True"
+    type: Ready
+```
+{{< /expand >}}
+
+## Reuse a patch
+
+A Composition can reuse a patch object on multiple resources with a 
+PatchSet.
+
+To create a PatchSet, define a 
+{{<hover label="patchset" line="5">}}PatchSets{{</hover>}} object inside the 
+Composition's
+{{<hover label="patchset" line="4">}}spec{{</hover>}}. 
+
+Each patch inside a PatchSet has a 
+{{<hover label="patchset" line="6">}}name{{</hover>}} and a list of
+{{<hover label="patchset" line="7">}}patches{{</hover>}}.  
+
+{{<hint "note" >}}
+For multiple PatchSets only use a single 
+{{<hover label="patchset" line="5">}}PatchSets{{</hover>}} object.  
+
+Identify each unique PatchSet with a unique 
+{{<hover label="patchset" line="6">}}name{{</hover>}}.
+{{</hint >}}
+
+Apply the PatchSet to a resource with a patch
+{{<hover label="patchset" line="16">}}type: PatchSet{{</hover>}}.  
+Set the 
+{{<hover label="patchset" line="17">}}patchSetName{{</hover>}} to the 
+{{<hover label="patchset" line="6">}}name{{</hover>}} of the PatchSet.
+
+```yaml {label="patchset"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+# Removed for brevity
+spec:
+  patchSets:
+  - name: my-patchset
+    patches:
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.desiredRegion
+      toFieldPath: spec.forProvider.region
+  resources:
+    - name: bucket1
+      base:
+        # Removed for brevity
+      patches:
+        - type: PatchSet
+          patchSetName: my-patchset
+    - name: bucket2
+      base:
+        # Removed for brevity
+      patches:
+        - type: PatchSet
+          patchSetName: my-patchset  
+```
+
+{{<hint "important" >}}
+A PatchSet can't contain other PatchSets.  
+
+Crossplane ignores any [transforms](#transform-a-patch) or
+[policies](#patch-policies) in a PatchSet.
+{{< /hint >}}
+
+## Patching between resources
+
+Compositions can't patch between resources in the same Composition. For example,
```

Comment:
This scenario actually can be done using bi-directional patching with the composed resources and a status field on the parent composite resource. I'll send you a link to a training deck that goes through that scenario ✅ 

There isn't a need to use 3 compositions here, this is done in platform-ref-aws specifically for modularity - the network resource can stand on its own without a cluster, so it needs to be its owns separate XR/Composition.

Reply:
This section has been rewritten using To/FromCompositeFieldPath patches.